src/store/flow/block/choice.ts

```diff
@@ -135,15 +137,15 @@ export const actions: ActionTree<IEmptyState, IRootState> = {
    * @param param1 action parameters
    * @returns
    */
-  choice_setSymonymForLanguage(
+  choice_setSynonymForLanguage(
     {getters, dispatch},
     {blockId, resourceId, language, index = Number.POSITIVE_INFINITY, value}: { blockId: IBlock['uuid'], resourceId: IResource['uuid'], language: ILanguage['id'], index: number, value: string },
   ) {
     const choice = getters.choice_findChoice(blockId, resourceId)
     const expressionValue = textValueToExpression(value)
 
     if (choice?.text_tests === undefined) {
-      choice.text_tests = []
+      Vue.set(choice, 'text_tests', [])
```

Comment:
@bulatgab , @object-required : This is the solution of the whole reactivity issues related to `choices. text_tests` 
When I think I spent 4h on this, 😆